### ISSUE TITLE
how to set epoch threshold?

### ISSUE THREAD

**Issue by BokyLiu** [Jianf-Wang/RSG#5]
Thank you for your nice work.
Could you share your experience about how to set epoch threshold?  
Does the threshold setting accord to cesc_loss or loss_mv_loss value?

**Comment by Jianf-Wang** [comment]
Hi, sorry for the late reply. In our experiments, we set the epoch threshold based on the learning rate. When the learning rate decreases to a small value, e.g., 0.001 (initial learning rate is 0.1),  RSG starts to generate new samples to adjust classifier.

**Comment by BokyLiu** [comment]
> accord to

thks, i will have a try in my dataset

**Comment by Jianf-Wang** [comment]
Plus, maybe you need to use large training epochs, especially before you decay the learning rate, as RSG needs more epochs to learn centers well.

**Comment by BokyLiu** [comment]
> Plus, maybe you need to use large training epochs, especially before you decay the learning rate, as RSG needs more epochs to learn centers well.

Yes, my training epoch is set to 300.  When the lr decreases under 0.001, i notice that the value of loss_cesc is around 2800. Does it normal?

**Comment by Jianf-Wang** [comment]
I think so. When I use RSG on CIFAR, the loss_cesc also does not further decrease, but it still works. RSG only estimates centers without changing and updating feature representations, so it is possible that loss_cesc does not further decrease.

**Comment by BokyLiu** [comment]
> I think so. When I use RSG on CIFAR, the loss_cesc also does not further decrease, but it still works. RSG only estimates centers without changing and updating feature representations, so it is possible that loss_cesc does not further decrease.

Wow, thks again!
